### DISCUSSION TITLE
Getting the error "No simulation found. Please add at least one simulation!"

### DISCUSSION THREAD

**Discussion by mercmobily** [ra3xdh/qucs_s#746] [Q&A] [answered]
I am trying to run a simulation.
This is my settings screen:

![image](https://github.com/ra3xdh/qucs_s/assets/2128734/7ee63022-2abd-49dd-8034-3dc5bc9251f8)

The simulator's output directory exists:

````
~$ ls -l /home/merc/.qucs/spice4qucs
total 8
-rw-rw-r-- 1 merc merc 292 Jun  9 23:45 spice4qucs.cir
-rw-rw-r-- 1 merc merc  88 Jun  9 23:45 spice4qucs.cir.dc_op
~$ 
````

ngspice runs:

````
~$ /usr/bin/ngspice
******
** ngspice-36 : Circuit level simulation program
** The U. C. Berkeley CAD Group
** Copyright 1985-1994, Regents of the University of California.
** Copyright 2001-2020, The ngspice team.
** Please get your ngspice manual from http://ngspice.sourceforge.net/docs.html
** Please file your bug-reports at http://ngspice.sourceforge.net/bugrep.html
** Creation Date: Mon Mar 11 21:44:53 UTC 2024
******
ngspice 1 -> 
````

What am I doing wrong?

**Comment by ra3xdh** [comment, marked as answer]
This question was asked and answered several times. Do literally what the error message says. Add at least one simulation on the schematic. The simulation is a special device that could be found in the left panel in the "simulation" group. This procedure described in a tutorial document. Please read this documentation. The link is available from the main page of the Qucs-S website: https://ra3xdh.github.io/

* Qucs-S tutorial https://ra3xdh.github.io/pdf/qucs_s_tutorial.pdf
* Video tutorial from Kasper Nielsen: https://www.youtube.com/watch?v=90RaVy38DB8
* Advanced simulation guide: https://habr.com/ru/articles/789452/ Sorry, only Russian version is available.

**Comment by ra3xdh** [comment]
I have transferred this issue to Discussions/Q&A. I would also advice you to upgrade the Ngspice package. The version 36 has many useful features missing.
